src/styles/blocks/cards.scss

```
@@ -0,0 +1,133 @@
+main {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, auto);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__items {
+    margin: 0 16px;
+
+    &--img {
+      height: 134px;
+      margin: 32px 19px 0;
+    }
+
+    &--description {
+      color: $main-color;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      margin-top: 40px;
+    }
+
+    &--code {
+      color: $card-secondary-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin-top: 4px;
+    }
+
+    &--stars-rating {
+      margin-top: 16px;
+      height: 16px;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+
+      .reviews {
+        color: $main-color;
+        text-align: right;
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
+        justify-content: flex-end;
+      }
+    }
+
+    &--price {
+      display: flex;
+      justify-content: space-between;
+      margin-top: 24px;
+    }
+
+    &--buy {
+      margin: 16px;
+      height: 40px;
+      border: 1px solid $secondary-color;
+      border-radius: 5px;
+      background: $secondary-color;
+      color: #fff;
+      text-align: center;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+      transition: background 0.3s ease, color 0.3s ease;
+
+      &:hover {
+        background: #fff;
+        color: $secondary-color;
+      }
+    }
+  }
+
+  &:hover .card__items--title {
+    color: #34568b;
+  }
+
+  .price {
```

remove it from nesting, it's a block
```suggestion
.price {
```